source/uri-options/uri-options.rst

```diff
@@ -50,7 +50,7 @@ occur:
    ``tlsDisableCertificateRevocationCheck`` appear in the URI options.
 7. Both ``tlsDisableOCSPEndpointCheck`` and
    ``tlsDisableCertificateRevocationCheck`` appear in the URI options.
-8. All instances of ``tls`` and ``ssl`` in the URI options do not have the
+8. Both ``tls`` and ``ssl`` appear in the URI options and they do not have the
```

Comment:
Can't comment on contextual code in a PR review, but the paragraph for this section states:
> Per the Connection String spec, the behavior of duplicates of most URI options is undefined.

This is no longer true, and should be removed entirely, leaving the paragraph as:
> Due to the security implications of certain options, drivers MUST raise an error to the user during parsing if any of the following circumstances occur: